src/pages/index.js

```diff
@@ -279,9 +213,49 @@ export default function IndexPage({ pkgVersion }) {
               }
             }}
           />
+          <div className="pb-6">
+            <SearchField
+              className="pt-6 w-full sm:px-6 md:px-0 md:mb-0"
+              inputClassName="w-full"
+              value={searchText}
+              onChange={(e) => {
+                const search = e.target.value;
+
+                setSearchText(e.target.value);
+
+                const filteredIcons =
+                  selectedTab !== 'all'
+                    ? getChromicons()?.filter((icon) =>
+                        icon?.categories?.includes(selectedTab)
+                      )
+                    : getChromicons();
+
+                if (!search) {
+                  setVisibleIcons(filteredIcons);
+                  return;
+                }
+
+                setVisibleIcons(
+                  filteredIcons?.filter(
+                    (icon) =>
+                      icon.rawName
+                        ?.toLowerCase()
+                        .includes(search.toLowerCase()) ||
+                      icon.name?.toLowerCase().includes(search.toLowerCase()) ||
+                      icon?.keywords
+                        ?.toLowerCase()
+                        ?.includes(search.toLowerCase())
+                  )
+                );
+              }}
+            />
+          </div>
         </div>
         {visibleIcons?.length > 0 ? (
-          <div className="flex flex-wrap justify-center px-4 my-4 max-w-6xl mx-auto sm:px-6 lg:px-16">
+          <div
+            className="flex flex-wrap justify-center mx-auto bg-white"
+            style={{ zIndex: '2' }}
```

Comment:
<img width="1491" alt="Screen Shot 2021-06-08 at 10 46 39 AM" src="https://user-images.githubusercontent.com/18708703/121207203-18d86480-c847-11eb-9332-2ddda074d104.png">
<img width="1503" alt="Screen Shot 2021-06-08 at 10 47 11 AM" src="https://user-images.githubusercontent.com/18708703/121207221-1d9d1880-c847-11eb-8391-282ae1b9fab0.png">

Reply:
This is the difference without the z-index

Reply:
Ah okay, nice.  Yeah one thing we may be able to do to clean up some of the z-indexes is to:

- Remove all of the existing z-index values set via `style`
- Add a negative z-index on the image that is offset

```jsx
<img
  src={`${process.env.BACKEND_URL}/big-icon.png`}
  className="absolute"
  alt="home banner"
  style={{
    bottom: -16,
    right: -5,
    top: -75,
    zIndex: -1, // new!
  }}
/>
```

- Add `bg-white` to the div (line 173) wrapped around `CategoryFilters`.  This will ensure the filter area has a white background as well
- Add `bg-transparent` onto the `main` tag.  This will allow the text to flow as it needs and ensures the background is transparent